### ISSUE TITLE
I want to translate the menu to Chinese, looking for help, thanks.

### ISSUE THREAD

**Issue by weidong0911** [OEFW-community/RT-890-custom-firmware#108]
Hello, my friends who share the same hobby and I really love your firmware, but since our mother tongue is Chinese, we hope to change the menu to Chinese. I have modified the target file app/menu.c with Chinese in the static const char Menu[][14] = {}, but after flashing, the Chinese characters did not show up. Could you give me some advice in your busy schedule? Since my foundation is weak, I am seeking your help, thank you very much.

**Comment by reppad** [comment]
Hi @weidong0911,
Thanks for your interest.
The problem is probably caused by the absence of Chinese font, you need to add or replace it by chinese fonts.

There is 3 fonts used in the firmware: regular (menus...), small (channel number, power...) and big that contains digits only (frequency when not in channel mode).
The small and big font are stored in firmware in file helper.c, you can replace it by chinese fonts and adapt the functions that draw characters.
The regular font is stored in flash, I don't know if the memory contains a Chinese font but it's not impossible, especially if there's a Chinese version of the Radtel RT-890 / Ruyage UV58Plus / Iradio UV-5118 Plus.
If there is, you can adapt the firmware to use it, if not you'll need to add it to an unused space of flash.
A simpler alternative would be to add it in firmware like small and big font but it will eat some space and you will need to disable another feature to avoid exceeding the available memory.

**Comment by weidong0911** [comment]
@reppad Thank you very much for your help! I am using the Iradio UV-5118 Plus because I have almost no experience. Following your suggestion, I spent an entire night asking AI for detailed steps to guide me through modifying the code, but it seems that AI is not intelligent, always giving wrong answers or precise answers but I don't know how to modify or add them. My efforts were unsuccessful, so I decided to give up! Anyway, thank you for your help!
